scripts/audit_frontend_licenses.py

```diff
@@ -49,6 +50,7 @@
     "Zlib",  # https://opensource.org/licenses/Zlib
     "Unlicense",  # https://unlicense.org/
     "WTFPL",  # http://www.wtfpl.net/about/
+    "MPL-2.0",  # https://www.mozilla.org/en-US/MPL/2.0/
```

Comment:
We should check with legal that this is ok - let's open a branch against `develop` that _just_ contains this change to the audit licenses script, and get legal review on it.